### PR TITLE
Implementation of Queue using stacks in Python

### PR DESCRIPTION
**Pull request by parulagg27**
**Fixes issue:**

#961 

**Changes:**

Added queue implementation with stacks in python.

### CODE REVIEW

File: code/data_structures/src/queue/queue_using_stack/queue_using_stack.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    def push(self, x):
+        self.items.append(x)
+
+    def pop(self):
+        return self.items.pop()
+
+    def size(self):
+        return len(self.items)
+
+    def isEmpty(self):
```
Comment (Vitorvgc) [inline code/data_structures/src/queue/queue_using_stack/queue_using_stack.py]: According to the style guide, functions should be named in lowercase separated by underscore

File: code/data_structures/src/queue/queue_using_stack/queue_using_stack.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+class Stack:
```
Comment (Vitorvgc) [inline code/data_structures/src/queue/queue_using_stack/queue_using_stack.py]: Missing header `Part of Cosmos by OpenGenus Foundation`

File: code/data_structures/src/queue/queue_using_stack/queue_using_stack.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    def __init__(self):
+        self.s1 = Stack()
+        self.s2 = Stack()
+
+    def enqueue(self, p):
+        self.s1.push(p)
+
+    def dequeue(self):
+        # pop all elements in s1 onto s2
+        while not self.s1.isEmpty():
```
Comment (Vitorvgc) [inline code/data_structures/src/queue/queue_using_stack/queue_using_stack.py]: Should replace to `is_empty()`

File: code/data_structures/src/queue/queue_using_stack/queue_using_stack.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            self.s2.push(self.s1.pop())
+
+        # pop all elements of s1
+        return_value = self.s2.pop()
+
+        # re pop all elements of s2 onto s1
+        while not self.s2.isEmpty():
```
Comment (Vitorvgc) [inline code/data_structures/src/queue/queue_using_stack/queue_using_stack.py]: And here also.

**Comment by parulagg27** [comment]
I totally skipped that function. Thanks for your review @Vitorvgc :)

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**